src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -410,123 +408,10 @@ export default class ResultsViewPage extends React.Component<
-                                            }
-                                            linkTooltip={
-                                                <div style={{ maxWidth: 400 }}>
-                                                    <a
-                                                        href="https://www.ndexbio.org/"
-                                                        target="_blank"
-                                                    >
-                                                        NDEx
-                                                    </a>{' '}
-                                                    shows 966 pathways:
-                                                    <ul>
-                                                        <li>
-                                                            211 from{' '}
-                                                            <a
-                                                                href="https://www.nlm.nih.gov/research/umls/sourcereleasedocs/current/NCI_PID/index.html"
-                                                                target="_blank"
-                                                            >
-                                                                NCI-PID
-                                                            </a>
-                                                        </li>
-                                                        <li>
-                                                            69 from{' '}
-                                                            <a
-                                                                href="https://signor.uniroma2.it/"
-                                                                target="_blank"
-                                                            >
-                                                                Signor
-                                                            </a>
-                                                        </li>
-                                                        <li>
-                                                            686 from{' '}
-                                                            <a href="https://www.wikipathways.org/">
-                                                                WikiPathways
-                                                            </a>
-                                                        </li>
-                                                    </ul>
-                                                </div>
-                                            }
-                                            hide={
-                                                !getServerConfig().show_ndex ||
-                                                !this.resultsViewPageStore
-                                                    .remoteNdexUrl.isComplete ||
```

Comment:
problem was that this actually gets executed even though the component which is consuming it won't render.  i.e. if you look at code around the tab, you will see that there is no condition which prevents this from getting executed.  you could put a condition activeTab=pathways, and that would also solve this problem, but really, the fetch for ndex status should be nested inside the pathways component so that it the code won't be reached unless the pathways tab actually renders.

Reply:
Gotcha -thank you! That makes sense